.gitignore

```diff
@@ -52,3 +52,6 @@ _output/
 
 # bin 
 bin/
+
+# patch
+*.patch
```

Comment:
@pohly brought this up in the kubernetes-csi standup meeting.  We think actually the .gitignore should only include files/binaries that are generated by this repository and not for files dependent on a dev's environment.

So for this particular repo, we should just ignore bin/

Reply:
for personal dev environments its possible to set up a local git ignore, this page has a guide for it: https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b